services/core-web/src/components/common/CoreTable.tsx

```diff
@@ -1,20 +1,20 @@
 import React from "react";
 import { Table, TableProps, Tooltip } from "antd";
-import { ColumnType } from "antd/es/table";
+import { ColumnsType } from "antd/es/table";
```

Comment:
The change to ColumnsType was just because of my struggles with type (see above) and attempt to fix it by getting rid of differences between what we had and antd demos. It didn't seem to affect anything.